packages/dds/tree/src/test/feature-libraries/optional-field/optionalField.spec.ts

```diff
@@ -76,7 +76,7 @@ const change1 = tagChangeInline(
 
 const change2Tag = mintRevisionTag();
 const change2: TaggedChange<OptionalChangeset> = tagChangeInline(
-	optionalFieldEditor.set(false, { fill: brand(42), detach: brand(2) }),
+	optionalFieldEditor.set(false, { fill: brand(42), detach: brand(2) }, tag),
```

Comment:
`tag` is not consistent with the revision that is passed to `tagChangeInline`
```suggestion
	optionalFieldEditor.set(false, { fill: brand(42), detach: brand(2) }, change2Tag),
```